packages/tasks/src/tasks/image-text-to-text/data.ts

```diff
@@ -0,0 +1,79 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Instructions composed of image and text.",
+			id: "liuhaotian/LLaVA-Instruct-150K",
+		},
+		{
+			// TODO write proper description
+			description: "Conversation turns where questions involve image and text",
+			id: "liuhaotian/LLaVA-Pretrain",
+		},
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "mask-generation-input.png",
+				type: "img",
+			},
+			{
+				label: "Text Prompt",
+				content: "Describe the position of the bee in detail.",
+				type: "text",
+			},
+		],
+		outputs: [
+			{
+				label: "Answer",
+				content: "The bee is sitting on a pink flower, surrounded by other flowers. The bee is positioned in the center of the flower, with its head and front legs sticking out.",
+				type: "text",
+			},
+		],
+	},
+	metrics: [],
+	models: [
+		{
+			description: "A powerful and grounded vision language model that can also localize concepts in images.",
+			id: "microsoft/kosmos-2-patch14-224",
+		},
+		{
+			description: "A strong vision language model that can also localize texts in images.",
+			id: "adept/fuyu-8b",
+		},
+        {
```

Comment:
```suggestion
		{
```

Reply:
I'll format locally once the content is approved, thanks though